spec/features/admin/admin_can_change_its_user_properties_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe "admin", type: :feature do
+
+  before :each do
+    @user = User.create(full_name:"Andrew Fink", email:"andrewfink@gmail.com", password:"password", role: 1)
+    visit login_path
+    fill_in('email', with: "andrewfink@gmail.com")
+    fill_in('password', with: "password")
+    click_button('come on in, farmhand')
+    visit edit_admin_user_path(@user)
+  end
+
+  it "can edit its user properties" do
+    expect(page).to have_button("update profile")
+  end
+
+  it "can update its full name" do
+    fill_in('full name', with: 'Douglas Froman')
+    click_button('update profile')
+
+    expect(current_path).to eq(admin_user_path(@user))
+    expect(page).to have_content('Douglas Froman')
+    expect(page).not_to have_content('Andrew Fink')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.